README.md

```diff
@@ -98,103 +98,115 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+ - Added CanIUse and TailwindCSS plugins mention to thirdPartyRunPlugins.md. Thanks [@skttl](https://github.com/skttl)!
+ - Added HttpStatusCodes plugin mention to thirdPartyRunPlugins.md. Thanks [@grzhan](https://github.com/grzhan)!
+ - Updated COMMUNITY.md with more contributors.
 
 ### Development
 
- - Fixed the CI precheck action to take into account the recent changes in CI actions.
- - Added the new Microsoft org issue types to the issue templates. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
- - Updated System.Text.Json to 8.0.5 and System.Runtime.Caching to 8.0.1 and related dependencies to the latest to address security reports. Thanks [@snickler](https://github.com/snickler)!
- - Updated WinAppSDK to 1.6.1 and CsWinRT to 2.1.5. Thanks [@snickler](https://github.com/snickler)!
- - Upgraded the WpfUI dependency to 3.0.5.
- - Updated MessagePack to 2.5.187 and StreamJsonRpc to 2.19.27 to address security reports.
- - Removed some of the hacks that are no longer needed that tried to force same dependency versions in .csproj files.
- - Removed the Markdown file exclusions from the conditions that trigger a full CI test.
- - CI fails again when there are XAML style errors in a PR.
- - Fixed CI actions that were not failing when one of the powershell scripts they tried to run was failing.
- - Fixed analyzer violations to allow fully building PowerToys on Visual Studio 17.12. Thanks [@snickler](https://github.com/snickler)!
-
-#### What is being planned for version 0.87
-
-For [v0.87][github-next-release-work], we'll work on the items below:
+ - Upgraded to .NET 9. Thanks [@snickler](https://github.com/snickler)!
+ - Fixed building on Visual Studio 17.12.
+ - Upgraded the System.IO.Abstractions dependency to 21.0.29. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - Upgraded the WindowsAppSDK dependency to 1.6.241114003. Thanks [@shuaiyuanxx](https://github.com/shuaiyuanxx)!
+ - Upgraded the MSTest dependency to 3.6.3. Thanks [@Youssef1313](https://github.com/Youssef1313)!
+ - Upgraded the check-spelling CI dependency to 0.0.24 and fixed related spell checking issues. Thanks [@jsoref](https://github.com/jsoref)!
+ - Removed duplicate names from the spellcheck allowed names file. Thanks [@htcfreek](https://github.com/htcfreek)!
+ - Improved logging of asynchronous methods call stacks when logging an error.
+ - Created a MSBuild props file to be imported by other projects to enable AOT support.
+ - Made the Peek utility source code AOT compatible.
+ - Updated .editorconfig rules to relax squiggly IDE errors in Visual Studio 17.12. Thanks [@snickler](https://github.com/snickler)!
+ - Moved Xaml.Styler from the root to the src folder.
+
+#### What is being planned for version 0.88
+
+For [v0.88][github-next-release-work], we'll work on the items below:
 
  - Stability / bug fixes
  - New module: File Actions Menu
```

Comment:
and ZoomIt?

Reply:
Yes, ZoomIt is still on the list. Hopefully on 0.88 😅

Reply:
Ah, got it, it appears on the line right after that one in the file, but won't appear on the diff unless you expand more lines.